Mathlib/Data/Set/Pointwise/Interval.lean

```diff
@@ -114,6 +114,41 @@ theorem Ici_mul_Ioi_subset' (a b : α) : Ici a * Ioi b ⊆ Ioi (a * b) := by
 
 end ContravariantLT
 
+section LinearOrderedCommMonoid
+variable [LinearOrderedCommMonoid α] [MulLeftReflectLE α] [ExistsMulOfLE α] {a b c d : α}
+
+@[to_additive (attr := simp)]
+lemma smul_Icc (a b c : α) : a • Icc b c = Icc (a * b) (a * c) := by
```

Comment:
Does this generalize to some arbitrary action type?

Reply:
It does, but I was hoping to not open this can of worms: we have `MulLeftMono` etc, but not corresponding `SMul` versions

Reply:
Let's go with a todo then?